.github/workflows/release.yaml

```diff
@@ -0,0 +1,35 @@
+name: Release
+
+on:
+  push:
+    tags:
+      - 'v[0-9]+.[0-9]+.[0-9]+'
```

Comment:
We need to support [pre-releases](https://semver.org/#spec-item-9)

Reply:
I think we can post-pone this until we actually need it, to be consistent across the falcosecurity.

Reply:
agree